clean.go

```diff
@@ -61,6 +61,13 @@ func isLicenseFile(path string) bool {
 	return licenseFilesRegexp.MatchString(path) && !isGoFile(path)
 }
 
+// vendorConfFilesRegexp is a regexp of file names which contain information about vendoring
+var vendorConfFilesRegexp = regexp.MustCompile(`vendor.conf`)
+
+func isVendorConfFile(path string) bool {
+	return vendorConfFilesRegexp.MatchString(path) && !isGoFile(path)
```

Comment:
Ah, sorry, I didn't read your commit message :) However, I still prefer to use exact match and not use other files for vendoring.